flocker/node/agents/cinder.py

```diff
@@ -515,9 +529,18 @@ def _blockdevicevolume_from_cinder_volume(cinder_volume):
     )
 
 
-@auto_openstack_logging(ICinderVolumeManager, "cinder_client.volumes")
-@auto_openstack_logging(INovaVolumeManager, "nova_client.volumes")
-@auto_openstack_logging(INovaServerManager, "nova_client.servers")
+@auto_openstack_logging(ICinderVolumeManager, "_cinder_volumes")
+class _LoggingCinderVolumeManager(PRecord):
+    _cinder_volumes = field(mandatory=True)
+
+@auto_openstack_logging(INovaVolumeManager, "_nova_volumes")
+class _LoggingNovaVolumeManager(PRecord):
+    _nova_volumes = field(mandatory=True)
+
+@auto_openstack_logging(INovaServerManager, "_nova_servers")
+class _LoggingNovaServerManager(PRecord):
+    _nova_servers = field(mandatory=True)
+
 def cinder_api(cinder_client, nova_client, cluster_id):
```

Comment:
Maybe it's just the late hour ... but ... I can't understand how this code could possibly have worked previously.  `auto_openstack_logging` is a class decorator but it was being used as a function decorator?  And a string with `.` in it was being used as an attribute name?

Was there any OpenStack logging enable at all prior to this change?